app/thirdparty/assets/prism/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
looks like some prism example, which we don't need. Would delete it and add it to .gitignore

Reply:
Yes, but it is convenient to test line numbers display appearance(e.g. line numbers fence padding) in browser instead of rebuilding, by custom the file /prism/plugins/line-numbers/prism-line-numbers.css.